### ISSUE TITLE
Feature Request: Publish prometheus-exporter-plugin to maven central.

### ISSUE THREAD

**Issue by adityachadha-tomtom** [Aiven-Open/prometheus-exporter-plugin-for-opensearch#289]
The OpenSearch project publishes all of the opensearch modules and Opensearch-project-provided plugins to [maven central](https://mvnrepository.com/artifact/org.opensearch.plugin/neural-search/2.15.0.0). 
This allows some users of opensearch to embed opensearch in their own JVM applications, loading plugins directly via custom dependency management and build tools e.g. maven, sbt, gradle, leiningen, ...; and injecting the plugin classes directly into OpenSearch Node instances.

would it be feasible to publish this plugin as an artifact to the [io.aiven org](https://mvnrepository.com/artifact/io.aiven) on mvnrepository? 

This would allow us a internally "compliant" way to take this to production.

**Comment by lukas-vlcek** [comment]
@adityachadha-tomtom Thanks for creating the ticket, I think it is duplication of https://github.com/Aiven-Open/prometheus-exporter-plugin-for-opensearch/issues/64 – that ticket has a long history in comments, just check out the latest comments at the bottom of the page to see the latest status.

I will close this ticket as a duplication.

**Comment by lukas-vlcek** [comment]
@adityachadha-tomtom Until this is fully implemented I think you should be able to setup temporary (internal) mvn repo. Would that help short-term?

**Comment by rursprung** [comment]
> [@adityachadha-tomtom](https://github.com/adityachadha-tomtom) Thanks for creating the ticket, I think it is duplication of [#64](https://github.com/Aiven-Open/prometheus-exporter-plugin-for-opensearch/issues/64) – that ticket has a long history in comments, just check out the latest comments at the bottom of the page to see the latest status.

i don't think that it's the same: my issue is specifically about publishing the ZIP file while this issue here seems to be about publishing the JAR files & co. (i.e. "normal" maven publishing of a java lib). the two issues are complementary (and when implementing #64 i guess this here comes more-or-less along for free)
